Use min_t() instead of min()

Fixes

| buffers.h:144:2: warning: implicit declaration of function 'min' [-Wimplicit-function-declaration]
| ...
| ../../libust/.libs/libust.so: undefined reference to `min'
| collect2: ld returned 1 exit status
| make[3]: *** [hello] Error 1


Index: ust-0.16/libust/buffers.h
===================================================================
--- ust-0.16.orig/libust/buffers.h
+++ ust-0.16/libust/buffers.h
@@ -141,7 +141,7 @@ extern void ltt_force_switch_lockless_sl
  */
 static inline unsigned int ltt_align(size_t align_drift, size_t size_of_type)
 {
-	size_t alignment = min(sizeof(void *), size_of_type);
+	size_t alignment = min_t(size_t, sizeof(void *), size_of_type);
 	return (alignment - align_drift) & (alignment - 1);
 }
 /* Default arch alignment */
